packages/queries/src/validators/validateNestedQueriesHaveAValidRefOp.ts

```diff
@@ -0,0 +1,29 @@
+import { QueryBuilderError } from '../QueryBuilderError';
+import { Query } from '../types/types';
+import { isRefOp } from '../types/isRefOp';
+
+/**
```

Comment:
Lots of grammar errors here.

```suggestion
Validate that every included sub-query has at least one refOp
```